File: pgm/ABC_Visual_Basic/ABC_139_B.vb
' Imports System.Collections.Generic
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim A, B As Integer
        Dim arr() As String
        arr = Split(Console.ReadLine())
        A = arr(0) : B = arr(1)
        Dim ans As Integer = 0
        DIm cur As Integer = 1
        While cur < B
            ans += 1
            cur += A - 1
        End While
        Console.WriteLine(ans)
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_139_B.cs
using System;
// using System.Collections.Generic;
// using System.Linq;
// using System.Text;

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        int A, B;
        A = int.Parse(arr[0]);
        B = int.Parse(arr[1]);
        int ans = 0;
        int cur = 1;
        while (cur < B)
        {
            ans++;
            cur += A - 1;
        }
        Console.WriteLine(ans);
        return;
    }
}